packages/examples-site/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Is this dependent on #21? Asking since that one is still in draft. I like that you added ts standards!

Reply:
Not dependant, but this file is a duplicate of the tsconfig in #21.